__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require(`../db/connection`);
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+	if (db.end) db.end();
+});
+
+describe('get:', () => {
+	describe('/api/topics', () => {
+		test('status:200 returns a single array', () => {
+			return request(app)
+				.get('/api/topics')
+				.expect(200)
+				.then(res => {
+					expect(res._body).toBeInstanceOf(Array);
+				});
+		});
+		test('status:200 returns a array of objects containing property: slug & description', () => {
+			return request(app)
+				.get('/api/topics')
+				.expect(200)
+				.then(res => {
+					res._body.forEach(news => {
+						expect(news).toEqual(
+							expect.objectContaining({
+								slug: expect.any(String),
+								description: expect.any(String),
+							}),
+						);
+					});
+				});
+		});
+		test('status:404 Not Found', () => {
+            return request(app).get('/api/topic').expect(404)
+                .then(res => {
+                    expect(res.error.path).toBe("/api/topic")
+                    expect(res.error.method).toBe("GET")
+                    expect(res.error.text).toBe("<!DOCTYPE html>\n<html lang=\"en\">\n<head>\n<meta charset=\"utf-8\">\n<title>Error</title>\n</head>\n<body>\n<pre>Cannot GET /api/topic</pre>\n</body>\n</html>\n")
```

Comment:
At the minute I believe this test is looking at the default error thrown back by express, check out app.all as a means of handling a "bad path" error manually, and sending back an object with a "msg" key of your choosing